pkg/cmd/server/api/helpers.go

```diff
@@ -187,13 +198,15 @@ func GetOpenShiftClient(kubeConfigFile string) (*client.Client, *kclient.Config,
 // for use by infrastructure components.
 func DefaultClientTransport(rt http.RoundTripper) http.RoundTripper {
 	transport := rt.(*http.Transport)
+	// TODO: this should be configured by the caller, not in this method.
 	dialer := &net.Dialer{
 		Timeout:   30 * time.Second,
 		KeepAlive: 30 * time.Second,
 	}
 	transport.Dial = dialer.Dial
 	// Hold open more internal idle connections
-	transport.MaxIdleConnsPerHost = 50
+	// TODO: this should be configured by the caller, not in this method.
+	transport.MaxIdleConnsPerHost = 100
```

Comment:
It includes watches.  If you have a hundred watches you can burst the limit and not reuse connections.  We have ~50 watches right now.

> On Jun 15, 2015, at 10:19 PM, Jordan Liggitt <notifications@github.com> wrote:
> 
> In pkg/cmd/server/api/helpers.go:
> 
> >  	dialer := &net.Dialer{
> >  		Timeout:   30 * time.Second,
> >  		KeepAlive: 30 * time.Second,
> >  	}
> >  	transport.Dial = dialer.Dial
> >  	// Hold open more internal idle connections
> > -	transport.MaxIdleConnsPerHost = 50
> > +	// TODO: this should be configured by the caller, not in this method.
> > +	transport.MaxIdleConnsPerHost = 100
> why boost this? what was gated by this?
> 
> —
> Reply to this email directly or view it on GitHub.
>